docs/source/user_guide/pipeline-components.md

```diff
@@ -395,36 +395,62 @@ Examples (CLI):
 
 The URL component catalog connector provides access to components that are stored on the web:
 - You can specify one or more URL resources.
-- The specified URLs must be retrievable using an HTTP `GET` request.
+- The specified URLs must be retrievable using an HTTP `GET` request. `http`, `https`, and `file` [URI schemes](https://www.iana.org/assignments/uri-schemes/uri-schemes.xhtml) are supported.
 - If the resources are secured, provide credentials, such as a user id and password or API key.
 
 Examples (GUI):
- - `https://raw.githubusercontent.com/elyra-ai/examples/main/component-catalog-connectors/kfp-example-components-connector/kfp_examples_connector/resources/filter_text_using_shell_and_grep.yaml`
+ - HTTPS URL
+   ```
+   https://raw.githubusercontent.com/elyra-ai/examples/main/component-catalog-connectors/kfp-example-components-connector/kfp_examples_connector/resources/filter_text_using_shell_and_grep.yaml
+   ```
+ - Local file URL
+   ```
+   file://local/path/to/component.yaml
+   ```
 
 Examples (CLI):
- - `['https://raw.githubusercontent.com/elyra-ai/examples/main/component-catalog-connectors/kfp-example-components-connector/kfp_examples_connector/resources/filter_text_using_shell_and_grep.yaml']`
- - `['<URL_1>','<URL_2>']`
+ - HTTPS URL
+   ```
+   ['https://raw.githubusercontent.com/elyra-ai/examples/main/component-catalog-connectors/kfp-example-components-connector/kfp_examples_connector/resources/filter_text_using_shell_and_grep.yaml']
+   ```
+ - Local file URL
+   ```
+   ['file://local/path/to/component.yaml']
```

Comment:
This is perhaps the biggest NIT of all time, but the connector will throw a 404 error if only 2 `/` are supplied

```suggestion
   ['file:///local/path/to/component.yaml']
```

Reply:
> This is perhaps the biggest NIT of all time

Not at all, great catch!  The third slash is the start of the absolute path and I wonder if, for clarity, the example URI should instead be:
```
['file:///absolute/path/to/component.yaml']
```
to emphasize this is an absolute path to the "Local file".

Reply:
I changed all occurrences of `file:...` in the doc to `file:///absolute/path/to/...`